internal/commands/scan_test.go

```diff
@@ -1187,3 +1192,40 @@ func TestValidateContainerImageFormat(t *testing.T) {
 		})
 	}
 }
+
+func TestRunScaResolverAndAddScaResultsFileCleanup(t *testing.T) {
```

Comment:
change the title to be more meaningfull. it hard to understand the test.
convention-
when_given_then(should)